src/pages/Game/Game.js

```diff
@@ -1,25 +1,79 @@
 import React, { Component } from "react";
 import { withStyles } from "@material-ui/core/styles";
-import Grid from "@material-ui/core/Grid";
-import GameData from "../../Classes/Game";
+import GameData from "../../Classes/GameData";
 import Player from "../../Classes/Player";
+import UpperStatusBar from "../../components/UpperStatusBar";
 
 const styles = theme => ({
-  root: {}
+  root: {
+    backgroundColor: "#718792",
+    minHeight: "200vh" // Temp
+  }
+});
+
+let game = new GameData({
+  id: 1,
+  adversary: new Player({ id: 1, tag: "Livvy", deck: [] }),
+  local: new Player({ id: 2, tag: "Phil714", deck: [] })
 });
 
 class Game extends Component {
+  constructor() {
+    super();
+    this.state = {
+      game: game
+    };
+
+    this.updateAdvHP = this.updateAdvHP.bind(this);
+    this.updateAdvMP = this.updateAdvMP.bind(this);
+    this.endTurn = this.endTurn.bind(this);
+    this.setAdvFaceSelect = this.setAdvFaceSelect.bind(this);
+  }
+
+  updateAdvHP(value) {
+    console.log("update hp");
+    game.adversary.hp += value;
+    this.setGame();
+  }
+
+  updateAdvMP(value) {
+    console.log("update mp");
+    game.adversary.mp += value;
+    this.setGame();
+  }
+
+  endTurn() {
+    console.log("end turn");
+  }
+
+  updateHand(value) {
+    console.log("update hand");
+    for (var i = 0; i < value; i++) game.adversary.hand.push("");
```

Comment:
Je pense que ça devrais être un int, à moins que tu veule mettre ses vrais cartes ?

Reply:
My problem is that the "player" class has an array for the hand. And I wanted it to be consistent. Like you have 2 players but one of them has an array of empty cards. Instead of one having an array and one have an int. I guess in JS I should really care about a variable being able to take on 2 different types but its bugging me still

Reply:
All that to say I'll see how Ill handle it when I plug the server in, right now those functions are just place holder / examples / ideas but they're probably not gonna stay as is